File: sdkwork-app-sdk-typescript/src/types/file-system-content-update-form.ts
/** Update file content request */
export interface FileSystemContentUpdateForm {
  /** Text content */
  text?: string;
  /** Multi-format content map */
  contents?: Record<string, string>;
  /** Prompt content */
  prompt?: string;
  /** Thinking content */
  thinkingContent?: string;
  /** Encoding */
  encoding?: string;
  /** Whether bumping file content version */
  bumpVersion?: boolean;
}

File: sdkwork-app-sdk-go/types/file_system_content_update_form.go
package types

// Update file content request
type FileSystemContentUpdateForm struct {
	Text string `json:"text"`
	Contents map[string]string `json:"contents"`
	Prompt string `json:"prompt"`
	ThinkingContent string `json:"thinkingContent"`
	Encoding string `json:"encoding"`
	BumpVersion bool `json:"bumpVersion"`
}
